### PR TITLE
Data Platform OMAS refactoring and fix some naming errors

### PR DESCRIPTION
**Pull request by cong78**
This PR is about update on fixing some naming errors, refactoring some classes and adapt to common services.

### CODE REVIEW

**Comment by planetf1** [comment]
The build failed with
```
07:24:43 [INFO] Building jar: /w/workspace/odpi-egeria-maven-verify-master-mvn35-openjdk8/open-metadata-implementation/access-services/data-platform/data-platform-server/target/data-platform-server-1.1-SNAPSHOT.jar
07:24:43 [INFO] 
07:24:43 [INFO] --- maven-dependency-plugin:3.1.1:analyze-only (analyze) @ data-platform-server ---
07:24:43 [WARNING] Used undeclared dependencies found:
07:24:43 [WARNING]    org.odpi.egeria:multi-tenant:jar:1.1-SNAPSHOT:compile
07:24:43 [INFO] Add the following to your pom to correct the missing dependencies: 
07:24:43 [INFO] 
07:24:43 <dependency>
07:24:43   <groupId>org.odpi.egeria</groupId>
07:24:43   <artifactId>multi-tenant</artifactId>
07:24:43   <version>1.1-SNAPSHOT</version>
07:24:43 </dependency>
```

So add the lines mentioned to your pom for data-platform-server -- but leave the `<version>` tag blank as it is driven by the top level pom `<dependencyManagement>`

If you still get an error (this time that it is unneeded), look for unnecessary imports. Consider use of intellij's 'optimize imports' option https://www.jetbrains.com/help/clion/optimizing-imports.html

A local `mvn clean install` should give the same result, but sometimes local contents of ~/.m2 can get in the way - so consider cleaning that repository/deleting the tree if seeing inconcistent results.

Also see https://www.jetbrains.com/help/clion/optimizing-imports.html

**Comment by cong78** [comment]
> The build failed with
> 
> ```
> 07:24:43 [INFO] Building jar: /w/workspace/odpi-egeria-maven-verify-master-mvn35-openjdk8/open-metadata-implementation/access-services/data-platform/data-platform-server/target/data-platform-server-1.1-SNAPSHOT.jar
> 07:24:43 [INFO] 
> 07:24:43 [INFO] --- maven-dependency-plugin:3.1.1:analyze-only (analyze) @ data-platform-server ---
> 07:24:43 [WARNING] Used undeclared dependencies found:
> 07:24:43 [WARNING]    org.odpi.egeria:multi-tenant:jar:1.1-SNAPSHOT:compile
> 07:24:43 [INFO] Add the following to your pom to correct the missing dependencies: 
> 07:24:43 [INFO] 
> 07:24:43 <dependency>
> 07:24:43   <groupId>org.odpi.egeria</groupId>
> 07:24:43   <artifactId>multi-tenant</artifactId>
> 07:24:43   <version>1.1-SNAPSHOT</version>
> 07:24:43 </dependency>
> ```
> 
> So add the lines mentioned to your pom for data-platform-server -- but leave the `<version>` tag blank as it is driven by the top level pom `<dependencyManagement>`
> 
> If you still get an error (this time that it is unneeded), look for unnecessary imports. Consider use of intellij's 'optimize imports' option https://www.jetbrains.com/help/clion/optimizing-imports.html
> 
> A local `mvn clean install` should give the same result, but sometimes local contents of ~/.m2 can get in the way - so consider cleaning that repository/deleting the tree if seeing inconcistent results.
> 
> Also see https://github.com/odpi/egeria/blob/master/developer-resources/Dependency-Management.md

Hi Nigel, thanks and i have fixed it by adding the dependency.

File: open-metadata-implementation/access-services/data-platform/data-platform-server/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/auditlog/DataPlatformAuditCode.java [hunk 1]
```diff
@@ -74,8 +74,73 @@
...
+            OMRSAuditLogRecordSeverity.ERROR,
+            "Entity matching criteria [{0}] could not be fetched. Error: {1}",
+            "The system is unable to process the request.",
+            "Verify the topic event."),
+    GET_RELATIONSHIP_EXCEPTION("OMAS-DATA-PLATFORM-0018",
+            OMRSAuditLogRecordSeverity.ERROR,
+            "Relationship {0} could not be fetched. Error: {1}",
+            "The system is unable to process the request.",
+            "Verify the topic event."),
+    BUILD_COLUMN_CONTEXT_EXCEPTION("OMAS-DATA-PLATFORM-0019",
```
Comment (ruxandraRosu) [inline open-metadata-implementation/access-services/data-platform/data-platform-server/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/auditlog/DataPlatformAuditCode.java]: I think some of these objects are not used, for example buildColumnContext is not in data platform flow

Comment (cong78) [reply open-metadata-implementation/access-services/data-platform/data-platform-server/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/auditlog/DataPlatformAuditCode.java]: Good point. :)

**Comment by ruxandraRosu** [comment]
#recheck

**Pull request merged**